### PR TITLE
Fix a few typos

### PR DESCRIPTION
**Pull request by hamlim**
A minor fix to the Readme to fix a few typos and minor grammatical updates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ perform a runtime assertion.
...
 Asynchronous tests could be challenging to get _right_, particulrary for junior
-developers or Engineers new to async JavaScript. The most common mistake is an async
+developers or engineers new to async JavaScript. The most common mistake is an async
 test which does not fire any assertions, either due to logic or even syntax errors.
 Static analysis(linters) gets close to pointing out the issues, but is not enough to catch logic mistakes.
 For this, we require a runtime check that _some_ assertion was ran during the test.
 
-[Jest, unfortunately, has not "failWithoutAssertions" configuration options, so this plugin aims to remedy that.]()
-The plugin patches Jest API to force tests without any assertions to fail. In addition
+[Jest, unfortunately, has no "failWithoutAssertions" configuration options, so this plugin aims to remedy that.]()
```
Comment (ballercat) [inline README.md]: Looks like I missed to actually add a proper link here, could you please fix it while you are here? Inside the parens for `to remedy that.]()` https://github.com/facebook/jest/issues/2209

**Review by ballercat — COMMENTED** [review]
I missed a closing doc comment on line 51, could you please add it? `*/`

File: README.md (hunk 1 above)
Comment (hamlim) [reply README.md]: Added 👍

**Comment by ballercat** [comment]
Thanks!

**Pull request merged**